string topic
---
string json_string
